src/survey.ts

```diff
@@ -763,9 +739,28 @@ export class SurveyModel extends SurveyElementCore
    */
   public onGetPanelFooterActions: EventBase<SurveyModel, GetPanelFooterActionsEvent> = this.addEvent<SurveyModel, GetPanelFooterActionsEvent>();
   /**
-   * Use this event to create/customize actions to be displayed in a matrix question's row.
+   * An event that allows you to add, delete, or modify actions in rows of a [Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/).
```

Comment:
Don't like an example. We can remove "Remove" action via another event by disable the remove.
It is better to show a new custom action.